### ISSUE TITLE
Create a basic server response in JSON format for testing

### ISSUE THREAD

**Issue by felipet** [felipet/lacoctelera_backend#2]
for example
https://jsonapi.org/

**Comment by felipet** [comment]
The change is ready when using 85dad6b. There's a new endpoint at `localhost:8080/ingredient` that returns an `Ingredient` entity formatted using JSON as follows:

```JSON
{
  "name":"Vodka",
  "category":"Spirit",
  "desc":"Voka liquour 40%"
}
```
@vera-zhurba let me know if this is enough for your tests.

**Comment by vera-zhurba** [comment]
@felipet  Hi! I got following error trying to fetch the data:
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://localhost:8080/ingredient. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing). Status code: 200.

I believe you can solve it. Thanks.

**Comment by felipet** [comment]
@vera-zhurba I've added the header `"Access-Control-Allow-Origin": "*"` to both GET & POST functions of the endpoint. The change is available in the commit that is lined a the top of this message. Check it out, and let me know if that fixes the issue.

**Comment by felipet** [comment]
The backend has to support the `OPTIONS` function for all the endpoints that expect a _complex http request_ (see [this](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#simple_requests).
As @vera-zhurba discovered, the browser sends an `OPTIONS` request before the actual request is even sent. If the response does not meet the browser's expectation, the actual request is not allowed to be issued to the server. 
A brief response to the `OPTIONS`  request would include:
- HTTP code 204
- The header `Access-Control-Allow-Headers`.
- The header `Access-Control-Allow-Origin`.

That means `Access-Control-Allow-Origin` is not only required in the responses to the `GET`/`POST` but for the response to `OPTION` request too. So far, none of the endpoints support such HTTP function. The `GET` for `/ingredient` does qualify as a _simple request_ but the `POST` doesn't, thus support for `OPTIONS ` of the endpoint `/ingredient` is mandatory to complete this task.

**Comment by vera-zhurba** [comment]
Well done, @felipet !  Everything works perfectly.  We can move on.

**Comment by felipet** [comment]
Merged into devel (356122e)
